src/lib/SX1280Driver/SX1280.cpp

```diff
@@ -307,21 +308,16 @@ void SX1280Driver::SetPacketParamsFLRC(uint8_t HeaderType,
     buf[3] = (uint8_t)syncWord;
 
     // DS_SX1280-1_V3.2.pdf - 16.4 FLRC Modem: Increased PER in FLRC Packets with Synch Word
-    if ((cr == SX1280_FLRC_CR_1_2) && ((buf[0] == 0x8C && buf[1] == 0x38) || (buf[0] == 0x63 && buf[1] == 0x0E)))
+    if (((cr == SX1280_FLRC_CR_1_2) || (cr == SX1280_FLRC_CR_3_4)) &&
+        ((buf[0] == 0x8C && buf[1] == 0x38) || (buf[0] == 0x63 && buf[1] == 0x0E)))
     {
         uint8_t temp = buf[0];
         buf[0] = buf[1];
         buf[1] = temp;
-    }
-    else
-    if (cr != SX1280_FLRC_CR_1_2)
-    {
-        // Not done for unused coding rates to save space.
-        ERRLN("!!! WARNING !!!");
-        ERRLN("Check datasheet errata for forbidden sync word bytes when using coding rates 3/4 and 1/1.");
-        ERRLN("!!! WARNING !!!");
-
-        while(1);
```

Comment:
The reason for this is so that if a dev adds a new coding rate later, then at least theres a message and the `while(1);` will cause a lockup/watchdog reset so it's obvious.

Reply:
Yeah I am aware of that buuuuuuuuuuuuut the amount of code it takes up for "Maybe someday!" with a comment like that is almost like it is meant to be a joke. The "save space" code 5x the size to actually do 100% support. I don't even think 100% support is needed because there's just 2 bits out of 32 that make a difference, and only if both of them are 0 which is what, 1 in 1-billion if they are random?

The comment just cracks me up though as the care given to taking 200 bytes of code to not work, compared to 2 if statements and a bit set and always working, while also Jedi Mind Tricking that it is better is just a gift to me. 😆

Reply:
Yeah, after I wrote it I thought it would probably take less space to write the code for the other CRs 😆

Reply:
Which is what I did and it was like 8 or 10 bytes! I just skipped that commented out bit because pfff 1 in a billion chance of 1 person's phrase maybe having reduced performance seemed like it didn't need more complexity.